.eslintrc

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Best practise is to extend an external ESLint config (as a dev dependency), or even better, just stick to [standard](feross/standard).